.travis.yml

```diff
@@ -1,11 +1,21 @@
 language: clojure
-lein: 2.8.1
-jdk:
-  - oraclejdk8
-script: ./ext/travisci/test.sh
-notifications:
-  email: false
```

Comment:
Are email notifications no longer enabled by default anymore?

Reply:
Oh, I'm not sure. I was just matching another project.

Reply:
I'd rather not have the email notifications on: https://docs.travis-ci.com/user/notifications/#default-notification-settings